libs/db/src/schema/socialMedia.ts

```diff
@@ -0,0 +1,11 @@
+import { integer, pgTable, serial, varchar } from 'drizzle-orm/pg-core'
+
+import { user } from './user'
+
+export const socialMedia = pgTable('socialMedia', {
+  id: serial('id').primaryKey(),
+  userId: integer('userId')
+    .references(() => user.id),
+  platformName: varchar('platformName', { length: 16 }),
```

Comment:
this should reference another `Platform` table so that we can regulate the social medias that we accept